perun-base/src/main/resources/perun-roles.yml

```diff
@@ -4395,52 +4427,70 @@ perun_policies:
       - FACILITYADMIN: Facility
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA: Facility
 
   unblockAllServicesOnDestination_int_policy:
     policy_roles:
       - FACILITYADMIN: Facility
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA: Facility
 
   unblockServiceOnFacility_Service_Facility_policy:
     policy_roles:
       - FACILITYADMIN: Facility
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA: Facility
 
   unblockServiceOnDestination_Service_int_policy:
     policy_roles:
       - FACILITYADMIN: Facility
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA: Facility
 
   forceServicePropagation_Facility_Service_policy:
     policy_roles:
       - FACILITYADMIN: Facility
       - ENGINE:
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA: Service
```

Comment:
same problem - Service objects don't have attributes, so we can't mark them as critical, thus it wouldn't make sense to have this check here